.yamllint.yaml

```
@@ -5,6 +5,9 @@ ignore: |
   *.param.yaml
 
 rules:
+  braces:
+    level: error
+    max-spaces-inside: 1
```

Can I add a comment?

```suggestion
    max-spaces-inside: 1 # To autoformat with Prettier
```